app/api/suggestions/specs/routes.spec.ts

```diff
@@ -35,6 +38,7 @@ jest.mock('api/services/informationextraction/InformationExtraction', () => ({
   },
 }));
 
+// eslint-disable-next-line max-statements
```

Comment:
Same here, if refactoring is not feasible, let's leave the error.

Reply:
Removing.